python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -290,9 +294,75 @@ def _fit(model, X_df_list, y_df_list, r):
         return model.fit(X_df, y_df)
 
     @staticmethod
-    def _predict(model, X, r):
+    def _predict_gpu(model, X_test, concat_mod_bytes):
+        preds = []
+        for i in range(len(X_test)):
+            X_test_df = X_test[i]
+            preds.append(model.predict(X_test_df,
+                                       concat_mod_bytes=concat_mod_bytes))
+        return preds
+
+    @staticmethod
+    def _predict_cpu(model, X, r):
         return model._predict_get_all(X)
 
+    @staticmethod
+    def _func_get_size(df):
+        return df.shape[0]
+
+    @staticmethod
+    def _func_get_idx(f, idx):
+
+        return f[idx]
+
+    @staticmethod
+    def _tl_model_handles(model, model_bytes):
+        return model._tl_model_handles(model_bytes=model_bytes)
+
+    def print_summary(self):
+        """
+        prints the summary of the forest used to train and test the model
+        """
+        c = default_client()
+        futures = list()
+        workers = self.workers
+
+        for n, w in enumerate(workers):
+            futures.append(
+                c.submit(
+                    RandomForestClassifier._print_summary,
+                    self.rfs[w],
+                    workers=[w],
+                )
+            )
+
+        wait(futures)
+        raise_exception_from_futures(futures)
+        return self
+
+    def concat_treelite_models(self, deep_check):
+        """
+        Convert the cuML Random Forest model present in different workers to
+        the treelite format and then concatenate the different treelite models
+        to create a single model. The concatenated model is then converted to
+        model bytes format.
+        """
+        mod_bytes = []
+        for w in self.workers:
+            mod_bytes.append(self.rfs[w].result().model_pbuf_bytes)
+
+        worker_numb = [i for i in self.workers]
+
+        list_mod_handles = []
```

Comment:
Variable names overall are kinda funny. Not an error per se, but some of them are overly verbose (list as a variable name for a list?) while others are abbreviated a little too much. Using the full word model would be better than having list here for example